src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -7,7 +7,7 @@
 
     /* Prefix definitions */
     public static final Prefix PREFIX_TITLE = new Prefix("title:");
-    public static final Prefix PREFIX_PHONE = new Prefix("p/");
+    public static final Prefix PREFIX_DATE = new Prefix("date:");
```

Comment:
KIV the prefix because this includes date and time

Reply:
the attribute in the userguide says date: tho